src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface StatusBadgeProps {
+    /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+    /** ClassName that appears in addition to "status-badge" */
+    className?: string;
+    /** Inner text of the element */
+    text?: string;
+    /** Type of badge **/
+    type?: string;
+}
+
+export default function Heading(props: React.PropsWithChildren<StatusBadgeProps>) {
+    const {
+        blockName,
+        className,
+        text,
+        type,
+    } = props;
+
+    let status_badge_base_class = "heading";
+
+    if (!props.children && !text) {
+        throw new Error("Status Badge has no children, please pass prop: text");
+    }
+
+    let statusBadgeModifiers = ["warning"];
```

Comment:
Looking at [Kathy's designs](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11256%3A0) maybe type and the modifiers associated with it could become levels—low, medium, high. I'm finding the warning/notice/etc. language confusing, especially as it doesn't explain which is more important than the other. What do you think?

Reply:
I agree.  I was going to rename them to "notice" and "alert", but as the design now has 3 levels, it makes more sense to be low, medium, high.  I will make that adjustment

Reply:
I think we should also change the prop name from `type` to `level` ? Type has a certain meaning in javascript that might make this confusing. What do you think? @scooooooooby @knezmilos

Reply:
I agree, good suggestion @nypl-wluisi.